devices.js

```diff
@@ -98,6 +98,28 @@ module.exports = {
       },
     },
   },
+  'lumi.sensor_magnet.aq2': {
+    name: 'Xiaomi Magnet Contact Sensor',
+    '@type': ['BinarySensor'],
```

Comment:
Do you thing we can use https://iot.mozilla.org/schemas/#DoorSensor here? If so, I think that `contact` property should be inverted and named `open`.

Reply:
Yeah that makes sense.